apps/system/style/window.css

```diff
@@ -337,9 +337,15 @@
   opacity: 1;
 }
 
+.appWindow.in-task-manager.overlay.no-screenshot > .identification-overlay {
+  visibility: visible;
+  opacity: 1;
```

Comment:
maybe we should add `transition: opacity 0.2s ease-in, visibility 0.2s;` to transition both ways in this case